chemie/rentalservice/templates/rentalservice/index.html

```diff
@@ -27,30 +67,30 @@
     </div>
   {% endif %}
 <div class="row">
-    {% for object in rentalObjects %}
-    <div class="col s12 m10 l6"> <!---size of the layout--->
-        <div class="card">
-            <div class="card-image">
-                <a href="{{ object.get_absolute_url }}">
-                    {% thumbnail object.image "630x400" crop="center" as im %} <!--- standard image crop for HC--->
-                        <img src="{{ im.url }}"> <!--- locates image url--->
-                    {% endthumbnail %}
-                </a>
-                <a class="card-title card-event-title" href="{{ object.get_absolute_url }}">
-                    <h4>{{ object.name }}</h4>
-                </a>
-            </div>
-            <div class="card-content">
-                <p class="card-text"> {{ object.description|safe }} </p>
-            </div>
-            <div class="card-action center">
-                <a href="{% url 'rentalservice:detail' object.id %}">Mer info</a> <!--- det er ingen hyperlenke her nå--->
+        <div class="col s12"> <!---size of the layout--->
+            <div class="row">
+            {% for object in rentalObjects %}
```

Comment:
I would prefer if the whole card was a hyperlink to the detail page, instead of just the text